### ISSUE TITLE
ES3 Json error on Debian 9 (Linux)

### ISSUE THREAD

**Issue by Stenodyon** [JimmyCushnie/TUNG#41]
Symptoms:
![screenshot_20180420_222739](https://user-images.githubusercontent.com/15184100/39072254-406c2bf0-44ea-11e8-84e2-80a6b2a25363.png)

Log file:
[Player.log](https://github.com/Iamsodarncool/TUNG/files/1933577/Player.log)

**Comment by JimmyCushnie** [comment]
Yeah, looks like ES3 ran into some data it couldn't parse. Should be fixed by deleting settings.txt.

I'm replacing ES3 with my own data text serialization library in 0.3, which should fix this.

**Comment by Stenodyon** [comment]
This was on a fresh install, tried removing settings.txt but I still get the same error.

This is what my [settings.txt](https://github.com/Iamsodarncool/TUNG/files/1935395/settings.txt) looks like (it's missing pretty much everything).

With a sane [settings.txt](https://github.com/Iamsodarncool/TUNG/files/1935406/settings.txt) this is my output log: [Player.log](https://github.com/Iamsodarncool/TUNG/files/1935407/Player.log)

**Comment by JimmyCushnie** [comment]
wow I really have no idea wtf is going on. I'm 90% sure it's not my fault, though - still all just ES3 errors...

**Comment by spjoe** [comment]
@Stenodyon 
When you have locales which represents a decimal point with , instead of . TUNG saves json which is invalid. So as a workaround delete "~/.config/unity3d/Mouse Hat Games/The Ultimate Nerd Game/" then start game with the following command: `LANG=C ./The\ Ultimate\ Nerd\ Game.x86_64`
@Iamsodarncool
Not sure which json lib you are using. I guess/hope you can hardcode culture.
So the actual problem already occurs when any json file is written (settings.txt, TimePlayed.txt, worldinfo.txt and maybe others).

It seems lib produces invalid json like `{ "a" : 1,2 }` but it should be  `{ "a" : 1.2 }`

**Comment by Stenodyon** [comment]
That was it, I can run the game properly now.
